coding-201-advanced-parsers/metaparser.py

```diff
@@ -0,0 +1,197 @@
+'''
+Metaparser
+----------
+   
+   Metaparser pyats module provides a structure unification among different
+   parsing contexts. It provides a unified structure that will be followed
+   by different device connections (Ex: 'cli', 'yang', 'xml).
+
+   It addresses the common automation problems as:
```

Comment:
It addresses common automation problems such as:

Reply:
Addressed.